### PR TITLE
nihal's fav food

### PR DESCRIPTION
**Pull request by nihalkarim**
added japanese curry chicken

@jag8932 @pkali076

### CODE REVIEW

File: favorite_foods.md [hunk 1]
```diff
@@ -8,3 +8,4 @@
 - Fish
 - Cucumber
 - Chicken Tenders
+- Japanese curry chicken
```
Comment (tstodter) [inline favorite_foods.md:11]: I didn't know you could grow curry chickens

Comment (jailynreese) [reply favorite_foods.md:11]: i love japanese curry

**Review by jailynreese — APPROVED** [review]

**Comment by pkali076** [comment]
close

**Pull request closed without merging**